telemetry/definitions/commonDefinitions.json

```diff
@@ -251,7 +256,7 @@
         {
             "name": "codeTransformApiNames",
             "type": "string",
-            "description": "Names of allowed API calls",
+            "description": "Deprecated (Names of allowed API calls)",
```

Comment:
ideally this results in deprecated annotations in the generated types, so that we can see this in the IDE